docker-compose.yaml

```diff
@@ -0,0 +1,41 @@
+version: '3.8'
+
+services:
+  postgres:
+    image: postgres:latest
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: tbd
+      POSTGRES_DB: mockpfi
+    ports:
+      - "5432:5432"
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  node:
```

Comment:
Since here you want to emulate a real deployment environment, instead of using `image: node:20.4.0` you should actually use your Docker image! 

example: https://github.com/TBD54566975/developer.tbd.website/blob/main/docker-compose.yaml#L4-L7

It will also simplify the command because you already have `dbmate`.

And it will force you to setup all the needed environment configs.

Reply:
@leordev are you suggesting that I replace the lines under `node` with an `app` section as shown in the example you linked?

And follow-up to that, where would the `command` section currently under `node` need to go? I don't think we got to the environment configs part in our discussion